### PR TITLE
Add event logging

### PR DESCRIPTION
**Pull request by andrewmarklloyd**
I'm not too familiar with Vue, but is this the best way to emit events from the AudioPlayer to the StickyPlayer? Any other potential issues with this?
We can also wait until #273 gets merged since we both made changes in LoginView. I can update this PR when that gets merged

### CODE REVIEW

File: src/components/StickyPlayer.vue [hunk 1]
```diff
@@ -40,7 +42,8 @@ export default {
   methods: {
     ...mapActions([
       'updatePlayerState',
-      'playEpisode'
+      'playEpisode',
+      'playEpisodeEvent'
     ]),
     onFinished () {
```
Comment (andrewmarklloyd) [inline src/components/StickyPlayer.vue]: I'd like to add a `completedEpisode` event here

**Comment by andrewmarklloyd** [comment]
We should figure out a way to only log events from the production server so we don't send events while testing

**Comment by andrewmarklloyd** [comment]
The accompanying [sedaily-devops PR](https://github.com/SoftwareEngineeringDaily/sedaily-devops/pull/67) will have to be merged and deployed before this PR can be merged. That PR adds a field to each event identifying the environment the event comes from. This will allow us to separate testing data and production data by filtering in our dashboards

File: config/apiConfig.js [hunk 2]
```diff
@@ -15,5 +15,6 @@ function getApiConfig() {
 }
 
 module.exports = {
-	apiConfig: getApiConfig()
+	apiConfig: getApiConfig(),
+	eventApiEnv: (process.env.EVENT_API_ENV == 'Prod') ? process.env.EVENT_API_ENV : 'Test'
```
Comment (abdulbasitkay) [inline config/apiConfig.js]: Shouldn't `Prod` be `production` as it's in line 3 of config/prod.env.js?

**Comment by andrewmarklloyd** [comment]
That's a good call. Updating it now

**Comment by andrewmarklloyd** [comment]
The [sedaily-devops PR](https://github.com/SoftwareEngineeringDaily/sedaily-devops/pull/67) is now merged and deployed. I've tested posting events locally.

@jasonify the only thing that this needs is an environment variable set on the production Heroku environment. When you have a chance can you set `EVENT_API_ENV=production`?

**Comment by andrewmarklloyd** [comment]
This is ready now.
Closed #355

File: src/components/StickyPlayer.vue (hunk 1 above)
Comment (abdulbasitkay) [reply src/components/StickyPlayer.vue]: That'll make a lot of sense

**Pull request merged**